src/notifier/slack_notifier.py

```diff
@@ -5,16 +5,19 @@
 import urllib.parse
 from typing import List
 
+# lib
+from confuse import ConfigView
+
 # project
 from . import Notifier, Event
 
 
 class SlackNotifier(Notifier):
-    def __init__(self, title_prefix: str, config: dict):
+    def __init__(self, title_prefix: str, config: ConfigView):
         logging.info("Initializing Slack notifier.")
         super().__init__(title_prefix, config)
         try:
-            credentials = config["credentials"]
+            credentials = config["credentials"].get()
```

Comment:
`get(dict)` to be consistent with others

Reply:
Yup, good catch.